stock_product_qty_by_packaging/models/stock_move.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Camptocamp SA
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# @author: Sébastien Alix <sebastien.alix@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import models
+
+
+class StockMove(models.Model):
+    _name = "stock.move"
+    _inherit = ["stock.move", "product.qty_by_packaging.mixin"]
+
+    _qty_by_pkg__qty_field_name = "product_uom_qty"
```

Comment:
shouldn't it be in the uom of the product ? Like for the move line
```suggestion
    _qty_by_pkg__qty_field_name = "product_qty"
```

Reply:
This didn't changed between 16.0 and 18.0, but that means it's a bug even in 16.0? And should be fixed in its own commit